doc/sphinxext/sphinxgallery/__init__.py

```diff
@@ -1,7 +1,7 @@
 """Sphinx Gallery
 """
 import os
-__version__ = '0.0.8'
+__version__ = '0.0.9-dev'
```

Comment:
I'm new to this release numbering. But the dev tag was on purpose so it shall not conflict with future versions on pypi.